### PR TITLE
Apply scm policy

### PR DESCRIPTION
**Pull request by yngve793**
Update in accordance with Equinor SCM policy.

Prepare README.md for project rename.

### CODE REVIEW

File: CONTRIBUTING.md
[the remark this answers is not in the document]
Comment (yngve793) [reply CONTRIBUTING.md:1]: It was suggested to add the standard SCM policy files before archiving the project. I am aware that several of these changes do not make sense at this point in time, but here we are. It is unclear to me how much work we should put into the project before archiving it.

**Review by yngve793 — COMMENTED** [review]
It was suggested to add the standard SCM policy files before archiving the project. I am aware that several of these changes do not make sense at this point in time, but here we are. It is unclear to me how much work we should put into the project before archiving it. 

I suggest to push your  "[Prototype] Version tests 2.0" and then the basic SCM before archive.

**Comment by achaikou** [comment]
Prototype PR should be closed I think. I hardly remember what it was about.

There is no such thing as standard SCM policy. Policy code that we submit should be fixed, at least factually incorrect things.
If we are archiving the project it doesn't mean that we can allow ourselves to push there anything incorrect.

Though we won't bother with fixing CI as this would actually require putting work into project which we don't need.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Contributing
+
+The following is a set of guidelines for contributing to `oneseismic`.
+
+There are several important ways you can help; here are some examples:
+
+- Submitting bug reports and feature requests: see [Issues](https://github.com/equinor/segyio/issues).
```
[the remark this answers is not in the document]
Comment (yngve793) [reply CONTRIBUTING.md]: Thanks.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-oneseismic
+oneseismic-old (Renamed from oneseismic)
 ==========
```
[the remark this answers is not in the document]
Comment (yngve793) [reply README.md]: Removed the comment for now. We can do this in separate commit.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+1. The commit headline should be descriptive and in the imperative
+
+
+## Pull request process
+
+1. Work on your own fork of the main repo.
+1. Push your commits and make a draft pull request.
+1. Check that your pull request passes all tests.
+1. When all tests have passed and your are happy with your changes, change your pull request to "ready for review", and ask for a code review.
+1. When your code has been approved you should rebase, squash and merge your changes.
```
[the remark this answers is not in the document]
Comment (yngve793) [reply CONTRIBUTING.md]: Ok

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Contributing
+
+The following is a set of guidelines for contributing to `oneseismic`.
+
+There are several important ways you can help; here are some examples:
```
[the remark this answers is not in the document]
Comment (yngve793) [reply CONTRIBUTING.md:5]: Yes, it is written by Matt so I presume that it is good.

**Review by yngve793 — COMMENTED** [review]
Fixed the raised issues. 
I am not sure if we want to merge non working code.

**Review by achaikou — APPROVED** [review]
Now we'll just have not to forget to fix here all the links to oneseismic before renaming!

**Pull request merged**